modules/media/README.md

```diff
@@ -1,51 +1,58 @@
-## Media Module
+# Media
 
-### 📄 Overview
+## Purpose
 
-Media module allows users to **upload**, **search** and **edit** media files associated with a specific candidate timepoint in Loris.
-Any kind of data associated with a candidate timepoint can be uploaded through this module: PDFs, videos, recordings, scripts, log files, etc. Files can optionally be associated to a specific instrument form within a given candidate timepoint.
+Media module allows users to **upload**, **search** and **edit** media files 
+associated with a specific candidate timepoint in Loris. Any kind of data 
+associated with a candidate timepoint can be uploaded through this module: 
+PDFs, videos, recordings, scripts, log files, etc. Files can optionally be 
+associated to a specific instrument form within a given candidate timepoint.
```

Comment:
I am happy to push these changes if you don't want to do it @ridz1208 ... but should this be "Files can optionally be associated WITH a specific instrument FROM within a given candidate timepoint." ?